src/components/Popover/Popover.module.css

```diff
@@ -0,0 +1,133 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # POPOVER
+\*------------------------------------*/
+
+/**
+ * 1) Box that contains content
+ */
+.popover {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-start;
+  visibility: hidden;
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(--eds-z-index-200) 0;
+  background-color: transparent;
+  transition: all var(--eds-anim-fade-long) var(--eds-anim-ease);
+  transform: translateZ(0);
+
+  &.eds-is-active {
+    visibility: visible;
+  }
+
+  &:focus-visible {
+    @mixin focus;
+  }
+}
+
+/**
+ * Popover window
+ * 1) Actual box with content within
+ */
+.popover__window {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: auto;
+  left: 0;
+  bottom: 4.25rem;
+  height: auto;
+  max-height: 28rem;
+  width: 25rem;
+  max-width: 25rem;
+  margin-left: 0.8125rem;
+  opacity: 0;
+  background: var(--eds-color-neutral-white);
+  border-radius: var(--eds-theme-border-radius);
+  box-shadow: var(--eds-theme-box-shadow);
+  transition: opacity var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  overscroll-behavior-y: contain;
+
+  &:focus-visible {
+    @mixin focus;
+  }
+
+  /**
+  * Popover arrow
+  * 1) A 0 height/0 width empty container positioned at the bottom of the popover with a ::after pseudo-element attached to it
+  * 2) Since the ::after pseudo element will use absolute positioning, it needs a parent that uses relative positioning; otherwise ::after will position itself relative to the page, not the popover
+  */
+  .popover__arrow {
+    position: relative; /* 2 */
+    bottom: 0;
+    left: 0;
+  }
+  /** Psuedo-element ::after
+  * 1) Adds a cosmetic triangle to the bottom edge of the popover, pointing toward the button that opened the popover
+  */
+  &::after {
+    position: absolute;
+    left: 1.25rem;
+    top: 100%;
+    border-top-color: var(--eds-color-neutral-white);
+    border-left: 0.9375rem solid transparent;
+    border-right: 0.9375rem solid transparent;
+    border-style: solid;
+    content: '';
+    border-width: 0.625rem;
+    border-bottom-width: 0;
+  }
+
+  /**
+   * Popover window inside active popover
+   * 1) Transform added for animation
+   */
+  .popover.eds-is-active & {
+    opacity: 1;
+    transform: translateY(0);
+    transition: opacity var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  }
+}
+
+/**
+* Popover content
+* 1) Includes header, body and footer of popover
+*/
+.popover__content {
+
+  height: auto;
+  max-height: 25rem;
+  overflow-y: auto;
+}
+
+/**
+ * Popover header
+ */
+.popover__header {
```

Comment:
Let's tighten up and even up the top and bottom padding of the Notification popover

Reply:
I figured out why my padding in the header was vanishing; I used `clsx` wrong on PopoverHeader.tsx:34. Fixed that and got the padding back.